src/data/employees.ts

```diff
@@ -584,6 +594,52 @@ const shreyaBhattarai: Employee = {
     linkedInLink: 'https://www.linkedin.com/in/shreyabha/',
 };
 
+const sushilTiwari: Employee = {
+    id: 'tc-072',
+    name: 'Sushil Tiwari',
+    position: 'Backend Developer',
+    description: 'Waasssssssssup Bro?',
```

Comment:
Are we sure the description is what Sushil sent?

Reply:
yes